earth_enterprise/src/third_party/gdal/SConscript

```diff
@@ -26,11 +26,28 @@
 # lines below.
 
 Import('third_party_env')
+import os
 
 gdal_version = 'gdal-2.1.2'
 openjpeg_version = 'openjpeg-v2.1.2-linux-x86_64'
 # kakadu_version = 'v5_2_6-00418C'
 # geoexpress_version = 'MrSID_DSDK-8.5.0.3422'
+
+# Check if MrSID decoding library is installed and get the decoding SDK path
+mrSIDLibName = 'libltidsdk.so'
+mrSIDDSDKPath = ""
+findLibraryCommand = "/sbin/ldconfig -p | grep %s | tr ' ' '\n' | grep /" %mrSIDLibName
+mrSIDlibraryFullPath=os.popen(findLibraryCommand).read()
+if not mrSIDlibraryFullPath:
+   print("mrSID library was not found ! GDAL will be built without mrSID raster format support.")
+else:
+   print "mrSID library was found (%s)! GDAL will be built with mrSID raster format support." %mrSIDlibraryFullPath
+   getFolderCommand = "dirname %s" %mrSIDlibraryFullPath
```

Comment:
Python includes os.path.dirname, though I'm not sure if it will do exactly the same thing.